### ISSUE TITLE
Upload: "+" to add multi-lingual captions difficult (impossible?) to find/use on certain screen sizes

### ISSUE THREAD

**Issue by PetitPotiron** [commons-app/apps-android-commons#5815]
When uploading an image, it is only possible to set the caption in 1 language. However, it would be great if we could set them for several languages, like on the new web upload wizard.

![The upload interface](https://github.com/user-attachments/assets/207179f7-b3e7-4855-bf09-703403e97ed9)

**Comment by nicolas-raoul** [comment]
This seems to be a layout issue.

On my screen when I tap the "+" button I can add captions in more languages:

![Screenshot_20240905-101448.png](https://github.com/user-attachments/assets/e9b0b957-97f4-4ef5-964f-69aa2092f8fe)

Any volumteer to make the UI elements reachable at any screen size?

**Comment by rohit9625** [comment]
Why the missing location icon is different in both screenshots?

**Comment by PetitPotiron** [comment]
Maybe it's the app version, I use the 4.2.1

**Comment by rohit9625** [comment]
> Maybe it's the app version, I use the 4.2.1

It's an old one. Try updating the app to the latest.

**Comment by mnalis** [comment]
yes, it is fixed and works in 5.1.2... I suggest closing the issue then.

**Comment by PetitPotiron** [comment]
Yes, I was about to say that yesterday. Thank you 👍
